lang/python/python-pycares/Makefile

```diff
@@ -0,0 +1,48 @@
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-pycares
+PKG_VERSION:=4.2.2
+PKG_RELEASE:=1
```

Comment:
depends on who you ask, you can also just do `PKG_RELEASE:=$(AUTORELEASE)`

i'm not sure how robust the mechanism is, but i use it;
it makes my life simple when updating a package, as there is one less line to edit; 

otherwise, the rule is: when updating to a new version, PKG_RELEASE gets reset to 1; otherwise, if something gets changed/updated, you need to bump PKG_RELEASE, which (as it sounds) is a pretty manual and (from experience) a bit of an error-prone process;

Reply:
Thanks, I'm happy to follow your lead here so updated accordingly next to the other suggestions